test/RequestClientWrapper.spec.ts

```diff
@@ -396,7 +396,7 @@ Code: 0 Message: You didnt pass any body`;
           .to.equal(`PUT ${Environment.STAGING}${requestUrl} -- STARTED`, 'should log correctly');
         done();
       });
-    });
+    }).timeout(5000);
```

Comment:
Why are you setting individual timeouts per test instead of just setting mocha opts to 5000? Seems cleaner to me to just let mocha opts handle it.

Reply:
@rjdavis3  I agree, however when mocha opts was set to 5000 Stryker took ~30 minutes to run, with the settings I have now it takes ~10 minutes. Would you still like me to change it? Maybe I can adjust the timers on the individual test so that they work within the timeout for the mocha opts?

Reply:
@csimko  wow that is a huge improvement. No faster builds are important so I am fine with this individual timeout. If there are other tests that can be improved this way just go for it. Ideally this build should only take a few minutes. I'll resolve this.

Reply:
@rjdavis3  I went ahead and just removed the .timeouts from those individual tests and adjusted the timing for them instead. There are no more .timeouts on any of the tests and they all timeout at the 1000ms mark which will keep Stryker from taking an absurd amount of time.